src/workerd/api/global-scope.c++

```diff
@@ -721,20 +721,14 @@ jsg::JsString ServiceWorkerGlobalScope::atob(jsg::Lock& js, kj::String data) {
   return js.str(decoded.asBytes());
 }
 
-void ServiceWorkerGlobalScope::queueMicrotask(jsg::Lock& js, v8::Local<v8::Function> task) {
-  auto fn = js.wrapReturningFunction(js.v8Context(),
-      JSG_VISITABLE_LAMBDA((this, fn = js.v8Ref(task)), (fn),
-          (jsg::Lock & js, const v8::FunctionCallbackInfo<v8::Value>& args)->v8::Local<v8::Value> {
-            return js.tryCatch([&]() -> v8::Local<v8::Value> {
-              auto function = fn.getHandle(js);
-
-              v8::LocalVector<v8::Value> argv(js.v8Isolate, args.Length());
-              for (int n = 0; n < args.Length(); n++) {
-              argv[n] = args[n];
-              }
-
-              return jsg::check(
-                  function->Call(js.v8Context(), js.v8Null(), argv.size(), argv.data()));
+void ServiceWorkerGlobalScope::queueMicrotask(jsg::Lock& js, jsg::Function<void()> task) {
+  auto fn = js.wrapSimpleFunction(js.v8Context(),
+      JSG_VISITABLE_LAMBDA((this, fn = kj::mv(task)), (fn),
+          (jsg::Lock& js, const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
```suggestion
          (jsg::Lock& js, const v8::FunctionCallbackInfo<v8::Value>&) {
```

If we're going to disregard a param in C++ we can leave it nameless.

Reply:
Will change that the next time I'm mucking around in this part of the code. It's not worth restarting CI over.

Reply:
it's been so slow today...